larq/optimizers.py

```diff
@@ -46,6 +46,24 @@
 __all__ = ["Bop", "CaseOptimizer"]
 
 
+def _var_key(var):
+    """Key for representing a primary variable, for looking up slots.
```

Comment:
might be nice to have a link to the source, in case that changes

Reply:
Good idea, I added a link. This function hasn't changed since TF 1.14, so there is a good chance that we can keep the implementation as is.